lib/fs.js

```diff
@@ -1793,43 +1778,12 @@ module.exports = fs = {
   writeSync,
   Stats,
 
-  get ReadStream() {
```

Comment:
An alternative fix is adding setters here to allow users to override the lazy-loaded variables so we can still avoid loading the `internal/fs/streams` module when requiring `fs`.

Reply:
That's how it's implemented now. It doesn't work with jest-mock because accessors are ignored.

Reply:
Ah right, I see that there are setters...so this looks more like a bug of Jest?

Reply:
Yes, maybe. But this was still kind of a sever major change and I'm not sure it's worth to do it.